src/Barricade.cpp

```diff
@@ -63,4 +63,4 @@ void Barricade::collidingProjectile(int damage) {
 void Barricade::placeBarricade(){
     texture.setAlpha(255);
     boolPlaced=true;
-}
+}
```

Comment:
removal of newline is very bad practice and in fact should be added where it is missing.
for reference:
http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file/18789
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
https://coderwall.com/p/icxxna/don-t-forget-that-extra-line